### PR TITLE
Ncc alloy v2 assets

### PR DESCRIPTION
**Pull request by struan**
This was partially reviewed earlier but never merged.

f3f65a7, e01cb6e and fd5eb18 are new.

[skip changelog]

### CODE REVIEW

File: test.log [hunk 1]
```diff
@@ -0,0 +1,44 @@
+2020/02/12-16:10:49 Starman::Server (type Net::Server::PreFork) starting! pid(36517)
```
Comment (dracos) [inline test.log:1]: Don't need to commit this? :)

File: web/cobrands/northamptonshire/assets.js [hunk 2]
```diff
@@ -253,9 +254,10 @@ fixmystreet.alloy_add_layers(northants_defaults, layers);
 
 // NCC roads layers which prevent report submission unless we have selected
 // an asset.
-var northants_road_defaults = $.extend(true, {}, fixmystreet.assets.alloyv2_defaults, {
+var northants_road_defaults = $.extend(true, {}, fixmystreet.alloyv2_defaults, {
     protocol_class: OpenLayers.Protocol.AlloyV2,
     http_options: {
+        base: "https://tilma.staging.mysociety.org/resource-proxy/proxy.php?https://northants.assets/${layerid}/${x}/${y}/${z}/cluster?styleIds=${styleid}",
```
Comment (dracos) [inline web/cobrands/northamptonshire/assets.js:260]: This should presumably use the is_live flag? Also presumably the "26" in the line below can't work, so does that want swapping with a placeholder or something? I see other commits have removed the need for is_live, is it still needed here? Ditto other places this changes.

File: web/cobrands/fixmystreet-uk-councils/alloy.js [hunk 3]
```diff
@@ -169,8 +167,9 @@ OpenLayers.Format.AlloyV2 = OpenLayers.Class(OpenLayers.Format.GeoJSON, {
 
 OpenLayers.Protocol.AlloyV2 = OpenLayers.Class(OpenLayers.Protocol.Alloy, {
     tileSize: 128,
+    resolution: 4.777314267158508,
```
Comment (dracos) [inline web/cobrands/fixmystreet-uk-councils/alloy.js:170]: Does this affect Hackney's use?

Comment (struan) [reply web/cobrands/fixmystreet-uk-councils/alloy.js:170]: Yes in as much as Hackney is likely to not work without this.

**Pull request closed without merging**